bloom-instance/tfvars.template

```diff
@@ -3,4 +3,5 @@ application_name = "Bloom Housing Instance"
 resource_prefix = "doorway_bloom"
 team_name = "doorway"
 aws_region = "us-west-1"
-sdlc_stage = "dev"
\ No newline at end of file
+sdlc_stage = "dev"
+vpc_cidr = "10.0.0.0/16"
```

Comment:
no idea if this is the right ip range...

Reply:
Yes, this is fine and a reasonable default.  We can use about anything for this value as long as it is a valid private CIDR block, it fits within the requirements for a VPC, and all our subnet CIDRs fall inside that range.

Reply:
ack